api/CustomEvent.json

```diff
@@ -152,24 +152,14 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/CustomEvent/initCustomEvent",
           "spec_url": "https://dom.spec.whatwg.org/#dom-customevent-initcustomevent",
           "support": {
-            "chrome": [
-              {
-                "version_added": true
-              },
-              {
-                "version_added": "59",
-                "notes": "<code>canBubble</code>, <code>cancelable</code>, and <code>detail</code> are optional parameters defaulting to <code>false</code>, <code>false</code>, and <code>null</code> respectively."
-              }
-            ],
-            "chrome_android": [
-              {
-                "version_added": true
-              },
-              {
-                "version_added": "59",
-                "notes": "<code>canBubble</code>, <code>cancelable</code>, and <code>detail</code> are optional parameters defaulting to <code>false</code>, <code>false</code>, and <code>null</code> respectively."
-              }
-            ],
+            "chrome": {
+              "version_added": "15",
+              "notes": "Since Chrome 59, <code>canBubble</code>, <code>cancelable</code>, and <code>detail</code> are optional parameters defaulting to <code>false</code>, <code>false</code>, and <code>null</code> respectively."
```

Comment:
I'm not sure what this note is about. Probably that the arguments were required before 59?

I'd like to remove the notes if possible :)